download.html

```diff
@@ -23,38 +23,34 @@
             </div>
     </div>
         </section>
-	<section id="configure">
-            <div class="text-center">
-                <h1>Configure</h1>
+
+<section id="requirements" class="container">
+    <div class="row ptb-15">
+        <div class="col-xs-12">
+            <h2>Requirements</h2>
+            <p>Gaffer has the following requirements:</p>
+            <h3>Operating System</h3>
+            <ul>
+                <li>Ubuntu or CentOS Linux</li>
```

Comment:
I originally wanted to put "Debian and RHEL." Thoughts?

Reply:
Do we even need to specify? Can we just say Linux?

Reply:
Since we haven't been too adventurous in seeing how tedious it would be to grab all the necessary packages in other, less media-rich distros, I figured it would be better to not over-promise. Maybe mention that these are the best-tested, or recommended, distributions?

Reply:
that makes sense to me